Cryptocurrency/Vertcoin/vtc-btc-bittrex_detail.30s.sh

```diff
@@ -0,0 +1,26 @@
+RESULT=$(curl -s "https://bittrex.com/api/v1.1/public/getmarketsummary?market=btc-vtc")
+vtcLast=$(echo $RESULT | egrep -o '"Last":\d+\.?\d{0,5}' | sed 's/"Last"://')
```

Comment:
```
vtcLast=$(echo $RESULT | egrep -o '"Last":\d+\.?\d{0,5}' | sed 's/"Last"://')
               ^-- SC2086: Double quote to prevent globbing and word splitting.
```